content/blog/2024-01-30-flash-framework.Rmd

```diff
@@ -0,0 +1,67 @@
+
+### Bridging the gap with FlaSH:
+
+Identifying and communicating noteworthy events from public health data is a critical step for future pandemic prevention <sup>[4](#link4)</sup>. But how should we redesign our event detection process? 
+After reflecting on our prior system, we realized the reason why existing monitoring fail, best summarized by the following quote: 
+
+<blockquote>
+"The current disconnect among algorithm developers, implementers, and [experts] … foster[s] distrust in statistical monitoring and in biosurveillance itself."  - Galit Shmeuli & Howard Burkom <sup>[5](#link5)</sup>
+</blockquote>
+    
+Interestingly, through our prior system, I had the opportunity to experience all three roles, from developing the outlier detection methods, implementing and deploying the methods over Delphi's data, and then analyzing the outputs of the methods. These three roles came with different constraints and priorities. These experiences set the foundation for devising an interdisciplinary approach. 
+
+<center>
+![**Fig 2.** FlaSH's Human-in-the-loop-Framework](/blog/2024-01-30-flash-framework/image2.png)</center>
+
+Combining the methods, engineering, and quality assurance perspectives for event detection from large-scale public health streams, we developed a human-in-the-loop approach to address some fundamental challenges in existing alerting systems. Namely, we rank Delphi's recently updated data by how much it warrants reviewer attention. Reviewers can then decide how many data points they want to investigate, which prioritizes a reviewer's time and does not expect them to review a certain number of alerts flagged via a somewhat arbitrary threshold. 
+
+Here's a quick illustration of why the ranking approach works. The most limited resource in this setting is an expert reviewer's time and attention. We want to show this reviewer as many possible events as possible so that they can interpret their data with their known context. 
+
+<center>
+![**Fig 3.** Illustration of Ranking vs. Alerts for large data streams](/blog/2024-01-30-flash-framework/image1.png)</center>
+
+Now imagine each data point per signal was given an outlier score, and there is an arbitrary binary threshold over all the data. If we develop granular enough rankings, we can directly compare outlier points between different signals without arbitrary thresholds. Then, a data reviewer can then focus on data points in their order of importance and stop the review at any time. 
```

Comment:
nits (also im not sure what you mean  by "arbitrary binary threshold") :
```suggestion
Now imagine each data point was given an outlier score, and there is an arbitrary binary threshold over all the data. If we develop granular enough rankings, we can directly compare outlier points between different signals more objectively. Then, our data reviewer can focus on data points in their order of importance and stop the review at any time. 
```

Reply:
This is supposed to show that if you set a threshold like 0.9 above which everything is an outlier and below which everything is not an outlier, that's a bit of an arbitrary number  (what about 0.89? why not 0.91?). So if you use rankings, you can combine these scores from different signals in a meaningful way because otherwise, there might just be a lot of things that are classified as outliers even if there is variation in those scores.

Reply:
Let me know if you think there's a clearer way to share this - I was hoping the picture would make it more clear, but if not this is the most important part of the blog, so maybe we can iterate on how to make it more understandable!

Reply:
@melange396 
Just wanted to double check on this part - does it make more sense now?

Reply:
it makes sense.  it was the word "binary" that threw me off; you mention thresholds a number of times before this, but this is the only time you call it a "binary threshold".